### PR TITLE
Initial config

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**
This PR configures the project and adds the proto and Java code moved from `base`.

### CODE REVIEW

**Comment by alexander-yevsyukov** [comment]
@armiol, PTAL.

File: money/build.gradle [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    api deps.build.guava
+    api group: 'io.spine', name: 'spine-base', version: spineBaseVersion
+
+    testCompile group: 'io.spine', name: 'spine-testlib', version: spineBaseVersion
+}
+
+modelCompiler {
+    generateValidatingBuilders = true
+}
+
```
Comment (armiol) [inline money/build.gradle]: Please kill this line.

File: money/src/main/proto/spine/money/money.proto [hunk 2]
```diff
@@ -0,0 +1,990 @@
...
+option java_package = "io.spine.money";
+
+option (experimental_all) = true;
+
+import "google/protobuf/descriptor.proto";
+
+//
+// The `Money` type below is modeled after one defined in Google APIs.
+// For the original code see:
+//   https://raw.githubusercontent.com/googleapis/googleapis/master/google/type/money.proto)
```
Comment (armiol) [inline money/src/main/proto/spine/money/money.proto]: There is an extra closing bracket at the end of this line.

File: money/src/main/proto/spine/money/money.proto [hunk 3]
```diff
@@ -0,0 +1,990 @@
...
+    AOA = 6 [(currency) = {
+        name: "Kwanza",
+        symbol: "Kz",
+        numeric_code: 973,
+		exponent_digits: 2 }];
+    ARS = 7 [(currency) = {
+        name: "Argentine Peso",
+        symbol: "$",
+        numeric_code: 32,
+		exponent_digits: 2 }];
```
Comment (armiol) [inline money/src/main/proto/spine/money/money.proto]: Some weird formatting here and there. I think the idea was to indent `numeric_code` and `exponent_digits` additionally to improve readability. But that's not done for all enumeration members.

**Review by armiol — APPROVED** [review]
@alexander-yevsyukov LGTM for the most parts. Please address the formatting issues before merging.

**Pull request merged**